ros/src/tl_detector/tl_detector.py

```diff
@@ -54,8 +52,21 @@ def __init__(self):
     def pose_cb(self, msg):
         self.pose = msg
 
+	# TODO move to image_cb 
+	# Get the light waypoint and state
+	light_wp, state = self.process_traffic_lights()
+	# If yellow or red publish waypoint
+	# Else publish -1
+	if state == 0 or state == 1:
```

Comment:
Let's use `TrafficLight.RED` and `TrafficLight.YELLOW` instead of the magic constants `0` and `1`.

Reply:
Also, we could extract the check into a method and reuse it in the `image_cb` method.